synapse/lib/storm.py

```diff
@@ -3516,10 +3521,28 @@ async def execStormCmd(self, runt, genr):
 
         if self.opts.tag:
 
-            tagname = await s_stormtypes.tostr(self.opts.tag)
+            tagnames = [await s_stormtypes.tostr(tag) for tag in self.opts.tag]
 
             layr = runt.snap.view.layers[0]
-            async for _, buid, sode in layr.liftByTag(tagname):
+
+            if len(tagnames) == 1:
+                async for _, buid, sode in layr.liftByTag(tagnames[0]):
+                    node = await self.runt.snap._joinStorNode(buid, {layr.iden: sode})
+                    if node is not None:
+                        yield node, runt.initPath(node)
+                return
+
+            genrs = [layr.liftByTag(tagname) for tagname in tagnames]
```

Comment:
Might be worth moving this down into the layer so we can merge before deepcopying the sodes

Reply:
I like it.  I also decided to remove the special-casing for len(tag)==1 since it just makes everything simpler and I don't think the overhead is that much.